Python/other/Maximum_Square_Matrix.py

```diff
@@ -0,0 +1,47 @@
+# find the size of Maximum Square Matrix With All 1s
```

Comment:
@rutujadhanawade Format your code according to PEP 8 rules. You can refer to this http://pep8online.com/.

Reply:
I have already formatted it. @plazzy99 
![python](https://user-images.githubusercontent.com/53823042/93891725-58870c80-fd09-11ea-9bb0-5f5dc9b57870.png)

Reply:
Done . Please have a look at it.